src/client/components/formComponent/formComponent.js

```diff
@@ -27,6 +30,11 @@ FormComponent.propTypes = {
   userData: PropTypes.instanceOf(Object).isRequired,
   onFormSubmit: PropTypes.func.isRequired,
   onInputChange: PropTypes.func.isRequired,
+  loading: PropTypes.bool.isRequired,
 };
 
-export default FormComponent;
+const mapStateToProps = state => ({
+  loading: state.authStatus.loading,
+});
+
+export default connect(mapStateToProps)(withRouter(FormComponent));
```

Comment:
Why do you need "withRouter" component here?

Reply:
dropped